### DISCUSSION TITLE
Transient simulation

### DISCUSSION THREAD

**Discussion by ICinoI** [e2nIEE/pandapipes#408] [General]
Hello,
Are there any plans to implement a transient solver into pandapipes ?

**Comment by jkisse** [comment]
Yes, it is intended to add transient heat transport simulation in the future and line pack behavior for gas networks. Would you be interested in participating in the development?

**Reply by ICinoI** [reply]
Thanks for your reply
Yes, I would be willing to participate to the best of my abilities.

**Reply by dlohmeier** [reply]
That sounds great. Could you let us know what you are interested in, gas grid simulation or rather heating grids? In the latter case I would like to point you to a quite new branch in our team repository that contains just a first draft for [transient heat transfer simulation](https://github.com/e2nIEE/pandapipes/tree/transient_heat_transfer). We are not currently working on it actively, but it is planned to do so in the near future. Probably also @Theda-ISE will join us there.

**Reply by tmkneiske** [reply]
I posted the information on the pandapipes Community event below. If want to join the kick-off, I can share the TEAMS link with you, just give me your email adresse and Name, Affiliation and sent it to tanja.kneiske@ieg.fraunhofer.de

**Reply by ICinoI** [reply]
I would be interested in participating in the gas grid simulation part.
I would also propose the development of a graphical user interface. I have already worked on implementing pandapipes into a simple GUI and i am willing to present and discuss it in the next pandapipes Community event.

**Comment by tmkneiske** [comment]
**Invitation to the „pandapipes Community” Kick-Off meeting** 

 

Date/Datum: 11. Oktober 2022,  11 am  

Place/Ort: online format, Invitation via TEAMS 

Target group/Zielgruppe: Everybody who has an idea which could be implemented with pandapipes./ Jeder, der eine Idee hat, die er mit pandapipes umsetzen möchte.  

Kontakt/Contact: [tanja.kneiske@ieg.fraunhofer.de](mailto:tanja.kneiske@ieg.fraunhofer.de) or [lars.lauven@iee.fraunhofer.de](mailto:lars.lauven@iee.fraunhofer.de)  

Anmeldung/Registration: If you want to participate, send an Email with your name, affiliation and project/topic to the Contacts. Einfach eine Email mit Name, Institut, Topic/Projekt an die Anmeldeemails senden. 

What? Was? 

We would like to carry out a first meeting as a workshop together with you. We would like to uncover common interests in applications of pipe flow simulation, discuss needed features and coordinate the further development of pandapipes. 

Wir möchten mit Euch einen Workshop veranstalten. Wir bieten an, gemeinsam mit Euch Synergien aufzudecken, Entwicklungen zusammen zu führen und die Entwicklungsaktivitäten zu koordinieren. 

Why? Warum? 

The pandapipes team would like to gather people who are working on and developing pandapipes in an open-source pandapipes-Community. We plan to organize a better exchange on new features and ongoing development and encourage a coordinated teamwork to develop necessary new modules and features of the pandapipes software. 

Das pandapipes Team möchte die pandapipes-Community näher zusammenbringen! Durch mehr Austausch hoffen wir, notwendige Entwicklungen unter Nutzern und Entwicklern besser koordinieren zu können. 

How? Wie? 

We would like to meet you in a Kick-Off workshop. Everybody can present their work and application of pandapipes. We will collect all your requirements and add them to a first pandapipes-Roadmap. Then we will present and discuss communication structures to work closer together in the future. This will include a proposal of a pandpaipes Governance structure. 

Wir stellen uns gegenseitig unsere Projekte vor. Wir fragen Eure individuellen Anforderungen ab und führen sie gemeinsam mit Euch in einer pandapipes – Roadmap zusammen. Wie finden wir geeignete Strukturen zum weitergehenden Austausch? 

What is not part of the Kick-Off? Was nicht? 

We will not do any pandapipes 101 or basic introduction of pandapipes. We will not have tutorials or will be able to answer detailed code questions. This will be done elsewhere. 

Keine Einführung für pandapipes Interessierte. Keine Vorstellung pandapipes Grundlagen für Neulinge. 

Seitenumbruch
 

 

Tentativ Agenda / Agendavorschlag (11th of October, online) 

 

11 to 11:15 am – 11 bis 11:15 Uhr 

Welcome, Intro and planning / Begrüßung, Einführung und Abstimmung Workshopablauf 

11:15 Uhr -13:00 Uhr (depending on the number of your contributions / je nach Teilnehmeranzahl) 

Introduction of participants / Vorstellungrunde (add coffee break / Kaffeepause einfügen) (Folien) 

Introduction of your project/application with pandapipes (if you like) / Vorstellung 1 Folie eigenes Projekt (jeder Teilnehmende der möchte) 

Give us a first idea of your requirements on pandapipes / Eure Anforderungen an pandapipes 

*break / Mittag* 

2 pm to 3pm / 14:00 Uhr – 15:00 Uhr  

ROADMAP- collecting requirements / Abfrage Anforderungen an eine Rohrnetzsimulationssoftware (Miro Board, all participants, we need you here!) 

Collect requirements / Sammeln der Anforderungen 

Cluster answers / Clustering der Antworten  

What can be done with pandapipes now and in the future / Bewertung: Was kann mit pandapipes umgesetzt werden? 

Priorisation of requirements / Zeitliche Einordnung der Anforderungen 

3pm tp 4 pm / 15:00 Uhr – 16:00 Uhr 

Collaboration and Governance / Abstimmung zur zukünftigen Zusammenarbeit (Folien und Miro Board)  

Presentation: What works so far and should be kept. / Vorstellung: Was bis jetzt funktioniert und beibehalten wird. (Simon) 

What do you need ? / Welche Bedarfe habt ihr? (Miro) 

Introduction to our Governance Structure / Vorstellung der Governance Struktur 

What are the next steps? / Was setzen wir als nächstes um?  

16:00 Uhr- 16:30 Uhr 

WrapUp etc. /sonst.

**Comment by tmkneiske** [comment]
Hi,

short answer „Yes“. It is the next important step to make.
But I cannot give you a certain timeline for that.

But we are planning an online workshop for pandapipes users to discuss further pandapipes development.
As a result we would like to generate a kind of roadmap.

So if you are interested let me now.

Tanja Kneiske


Von: ICinoI ***@***.***>
Gesendet: Dienstag, 13. September 2022 18:59
An: e2nIEE/pandapipes ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [e2nIEE/pandapipes] Transient simulation (Discussion #408)


Hello,
Are there any plans to implement a transient solver into pandapipes ?
